src/main/java/appeng/client/render/DelegateBakedModel.java

```diff
@@ -0,0 +1,52 @@
+
+package appeng.client.render;
+
+
+import javax.vecmath.Matrix4f;
+
+import org.apache.commons.lang3.tuple.Pair;
+
+import net.minecraft.client.renderer.block.model.IBakedModel;
+import net.minecraft.client.renderer.block.model.ItemCameraTransforms;
+import net.minecraft.client.renderer.texture.TextureAtlasSprite;
+
+
+public abstract class DelegateBakedModel implements IBakedModel
+{
+	private IBakedModel baseModel;
+
+	protected DelegateBakedModel( IBakedModel base )
+	{
+		this.baseModel = base;
+	}
+
+	@Override
+	public Pair<? extends IBakedModel, Matrix4f> handlePerspective( ItemCameraTransforms.TransformType type )
+	{
+		Pair<? extends IBakedModel, Matrix4f> pair = this.baseModel.handlePerspective( type );
+		return Pair.of( this, pair.getValue() );
+	}
+
+	@Override
+	public TextureAtlasSprite getParticleTexture()
+	{
+		return this.baseModel.getParticleTexture();
+	}
+
+	@Override
+	public ItemCameraTransforms getItemCameraTransforms()
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Don't override a deprecated method or explicitly mark it as "@Deprecated". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)